pages/stories.tsx

```diff
@@ -1,11 +1,33 @@
 import type { NextPage } from 'next'
+import React from 'react';
+import ImageGallery from 'react-image-gallery';
+
+const IMAGES = [
+  {
+    original: 'https://picsum.photos/id/1018/1000/600/',
```

Comment:
Can you update these links to some actual links to manuscript images, until we can start pulling from the db?

Reply:
You can also just change it in your `carousel` branch and just push up the changes. It will automatically update this Pull Request instead of having to create a whole new one.